docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
 
-```py
-from cartoframes.data import enrichment
-from cartoframes.viz.helpers import color_continuous_layer
 
-arlington_no_car_df = enrichment.enrich_polygons(arlington_tracts, acs_no_car, {'no_car': 'SUM'}, data_geom_column='geometry_wkt')
-Map(color_continuous_layer(arlington_no_car_df, value='no_car', title='No cars households'))
+
+    [<Geography('ags_blockgroup_1c63771c')>,
+     <Geography('ags_q17_4739be4f')>,
+     <Geography('mbi_blockgroups_1ab060a')>,
+     <Geography('mbi_counties_141b61cd')>,
+     <Geography('mbi_county_subd_e8e6ea23')>,
+     <Geography('mbi_pc_5_digit_4b1682a6')>,
+     <Geography('usct_blockgroup_f45b6b49')>,
+     <Geography('usct_cbsa_6c8b51ef')>,
+     <Geography('usct_censustract_bc698c5a')>,
+     <Geography('usct_congression_b6336b2c')>,
+     <Geography('usct_county_ec40c962')>,
+     <Geography('usct_place_12d6699f')>,
+     <Geography('usct_puma_b859f0fa')>,
+     <Geography('usct_schooldistr_515af763')>,
+     <Geography('usct_schooldistr_da72a4cb')>,
+     <Geography('usct_schooldistr_287be4f7')>,
+     <Geography('usct_state_4c8090b5')>,
+     <Geography('usct_zcta5_75071016')>]
+
+
+
+This time, let's choose the block groups from AGS and check which datasets are avaialble.
+
+
+```python
+Catalog().country('usa').category('demographics').geography('ags_blockgroup_1c63771c').datasets
 ```
 
-![Bikeshare data](../img/new-guides/quickstart/enrich_helper.png)
 
-Nice! Thanks to our layer helper, we can already see which are the areas with the highest percentage of households with no cars. You can learn more about discovering and enriching your data in the [data guide]().
 
-### Analyse if the current bike stations are placed in optimal locations
 
-We can already suggest which are the areas where can make more sense to have a station looking at the ones that have more households with no cars. In this step, let's try to go a bit further and try to calculate which areas have significantly high or low numbers of them. We will use the open source library [pySAL](https://pysal.org) for that.
+    [<Dataset('ags_sociodemogr_e92b1637')>,
+     <Dataset('ags_consumerspe_fe5d060a')>,
+     <Dataset('ags_retailpoten_ddf56a1a')>,
+     <Dataset('ags_consumerpro_e8344e2e')>,
+     <Dataset('ags_businesscou_a8310a11')>,
+     <Dataset('ags_crimerisk_9ec89442')>]
+
 
-You decide to use [Moran's I]() to measure the spatial autocorrelation and [Queen]() as the algorithm to decide which areas are considered neighbours. Once you are done with your analysis (here we are just showing a simplified version of it), you assign a label with the significance level to each station.
 
-```py
-from libpysal.weights import Queen
-from pysal.explore.esda.moran import Moran_Local
+Nice! The population variables should be inside the sociodemographic dataset, let's find them looking into their descriptions.
```

Comment:
replace: "Nice! The population variables should be inside the sociodemographic dataset, let's find them looking into their descriptions."
with: "Nice! The population variables are inside of the sociodemographic category, let take a look at what options are available and the associated descriptions."